servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -2427,6 +2428,8 @@ void RendererCanvasRenderRD::_record_item_commands(const Item *p_item, RenderTar
 				instance_data->dst_rect[2] = dst_rect.size.width;
 				instance_data->dst_rect[3] = dst_rect.size.height;
 
+				instance_data->instance_uniforms_ofs = uint32_t(p_item->instance_allocated_shader_uniforms_offset);
```

Comment:
This should be passed in as an argument to `new_instance_data()` so that it is always initialized. Right now this will only impact certain canvas_items (like Sprite2Ds)

Reply:
Should it still be set here?